docs/reference/service.rst

```diff
@@ -21,6 +21,50 @@ Renku Core Service
 The Renku Core service exposes a functionality similar to the Renku CLI via a
 JSON-RPC API.
 
+Components:
+
+.. uml::
```

Comment:
why uml and not mermaid?

Reply:
It's even uglier in mermaid. mermaid just doesn't really support this kind of diagram well and doesn't offer any control over layout. I tried for 1-2 hours and couldn't get a good result with mermaid. 

Since we use plantuml in the main `renku` repo, I decided to check that out, and [this](https://plantuml.com/component-diagram) looked much closer to what I was going for. And I can specify the cardinal direction of arrows at least. It even follows what I specify roughly 70% of the time!

Reply:
I was hoping to harmonize all the diagrams to mermaid so we only ever have to use one...

Reply:
That's ok for diagram types mermaid supports, but they don't support that many. You can thumbs-up https://github.com/mermaid-js/mermaid/issues/1253 if you'd like. That's what I'd really want. Mermaid is fine when layout doesn't matter or if you're lucky and the layout is fine, but the lack of control over the layout is a big hindrance imo.

The alternative is I make it with draw.io exactly how I want and store it as svg.

Reply:
ok I see what you mean - I like this one, we can beautify it later :)